Documented current Private Cloud limitations (#2511)
mendix/docs: content/developerportal/deploy/private-cloud-supported-environments.md

{{% alert type="info" %}}
For every Mendix app environment, a new bucket, IAM user and inline policy will be created so that the app can only access its own bucket.
{{% /alert %}}

To use S3, the Mendix Operator will need an IAM account with the following policy so that it can create a new IAM user and bucket for each Mendix app environment:

```json
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "bucketPermissions",
            "Effect": "Allow",
            "Action": [
                "s3:CreateBucket",
                "s3:DeleteBucket"
            ],
            "Resource": "arn:aws:s3:::mendix-*"
        },
        {
            "Sid": "iamPermissions",
            "Effect": "Allow",
            "Action": [
                "iam:DeleteAccessKey",
                "iam:PutUserPolicy",
                "iam:DeleteUserPolicy",
                "iam:DeleteUser",
                "iam:CreateUser",
                "iam:CreateAccessKey"
            ],
            "Resource": [
                "arn:aws:iam::<account_id>:user/mendix-*"
            ]
        }
    ]
}
```

## 6 Networking

{{% alert type="info" %}}

> DNS, load balancing and the ingress controller should be configured separately, on a cluster-wide scope.

Reviewer: @zlogic Do you mean that you have to configure them outside the Mendix scope, or do you mean that you need to configure each of the three separately? If the former then perhaps:
```suggestion
DNS, load balancing and the ingress controller should be configured first for the whole Kubernetes cluster.
```

Author: @MarkvanMents Your understanding is correct, it's outside of the Mendix scope and is the customer's responsibility.
